doc/latex.rst

```diff
@@ -1294,10 +1307,17 @@ forget the underscore separating the prefix from the property names.
   * or two dimensions followed by the keyword ``inset``.
 
   The x-offset and y-offset may be negative.  The default is ``none``,
-  *except* for the :dudir:`topic` or contents_ directives, for which it is
-  ``4pt 4pt``, i.e. the shadow has a width of ``4pt`` and extends to the right
-  and below the frame.  The lateral shadow then extends into the page right
-  margin.
+  *except* for:
+
+  - the contents_ directive: ``4pt 4pt``, i.e. the shadow has a width of
```

Comment:
arrgghh... I am starting to think we should have used `longfbox` whose [doc](https://ctan.org/pkg/longfbox) already has nice pictures (sadly the package is broken for years because it was developed at a time `pict2e` defined a `\@tempdimd` but later dropped it) :smiley:.  Well, doesn't [CSS documentation sites](https://developer.mozilla.org/fr/docs/Web/CSS/box-shadow) for HTML fit the bill already? The behavior of `box-shadow` is directly emulating it, with  a more restricted syntax, but the way "inset" keyword or negative dimensions impact result should be exactly as in CSS.  At least this is what I remember from when this was done.